.github/workflows/bundleTest-thor.yml

```diff
@@ -0,0 +1,276 @@
+name: BundleTest on Thor
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+        description: 'Operating System'
+        required: false
+        default: 'ubuntu-22.04'
+      asset-name:
+        type: string
+        description: 'Asset Name'
+        required: false
+        default: 'build-docker-package'
+      dependencies:
+        type: string
+        description: 'Dependencies'
+        required: false
+        default: 'bison flex build-essential binutils-dev curl lsb-release libcppunit-dev python3-dev default-jdk r-base-dev r-cran-rcpp r-cran-rinside r-cran-inline pkg-config libtool autotools-dev automake git cmake xmlstarlet'
+      get-Stat: 
+        type: boolean
+        description: 'Run Query stat'
+        required: false
+        default: false
+      generate-Zap:
```

Comment:
recommend you use all lower case just for consistency when you have variable names like this